q2_feature_table/plugin_setup.py

```diff
@@ -509,8 +508,9 @@
 
 plugin.visualizers.register_function(
     function=q2_feature_table.tabulate_seqs,
-    inputs={'data': FeatureData[Sequence | AlignedSequence]},
-    parameters={},
+    inputs={'data': FeatureData[Sequence | AlignedSequence],
+            'taxonomy': FeatureData[Taxonomy]},
+    parameters={'metadata': Metadata},
```

Comment:
We'll want to write descriptions for these

Reply:
Maybe for taxonomy go with "The taxonomic classifications of the tabulated features." and for the metadata something like "Any additional metadata for the tabulated features."